GVRf/Framework/jni/engine/importer/assimp_importer.cpp

```diff
@@ -68,4 +68,428 @@ Mesh* AssimpImporter::getMesh(int index) {
+        jobject& new_instance) {
+    jclass java_class = env->FindClass(class_name);
+
+    if (NULL == java_class) {
+        return false;
+    }
+
+    jmethodID constructor_id = env->GetMethodID(java_class, "<init>", "()V");
+
+    if (NULL == constructor_id) {
+        return false;
+    }
+
+    new_instance = env->NewObject(java_class, constructor_id);
+    env->DeleteLocalRef(java_class);
+
+    if (NULL == new_instance) {
+        return false;
+    }
+
+    return true;
+}
+
+bool AssimpImporter::create_instance(JNIEnv *env, const char* class_name,
+        const char* signature,/* const*/jvalue* params, jobject& new_instance) {
+    jclass java_class = env->FindClass(class_name);
+
+    if (NULL == java_class) {
+        return false;
+    }
+
+    jmethodID constructor_id = env->GetMethodID(java_class, "<init>",
+            signature);
+
+    if (NULL == constructor_id) {
+        return false;
+    }
+
+    new_instance = env->NewObjectA(java_class, constructor_id, params);
+    env->DeleteLocalRef(java_class);
+
+    if (NULL == new_instance) {
+        return false;
+    }
+
+    return true;
+}
+
+bool AssimpImporter::get_field(JNIEnv *env, jobject object,
+        const char* field_name, const char* signature, jobject& field) {
+    jclass java_class = env->GetObjectClass(object);
+
+    if (NULL == java_class) {
+        return false;
+    }
+
+    jfieldID field_id = env->GetFieldID(java_class, field_name, signature);
+    env->DeleteLocalRef(java_class);
+
+    if (NULL == field_id) {
+        return false;
+    }
+
+    field = env->GetObjectField(object, field_id);
+
+    return true;
+}
+
+bool AssimpImporter::set_object_field(JNIEnv *env, jobject object,
+        const char* field_name, const char* signature, jobject value) {
+    jclass java_class = env->GetObjectClass(object);
+
+    if (NULL == java_class) {
+        return false;
+    }
+
+    jfieldID field_id = env->GetFieldID(java_class, field_name, signature);
+    env->DeleteLocalRef(java_class);
+
+    if (NULL == field_id) {
+        return false;
+    }
+
+    env->SetObjectField(object, field_id, value);
+
+    return true;
+}
+
+bool AssimpImporter::call(JNIEnv *env, jobject object, const char* type_name,
+        const char* method_name, const char* signature,/* const*/
+        jvalue* params) {
+    jclass java_class = env->FindClass(type_name);
+
+    if (NULL == java_class) {
+        return false;
+    }
+
+    jmethodID method_id = env->GetMethodID(java_class, method_name, signature);
+    env->DeleteLocalRef(java_class);
+
+    if (NULL == method_id) {
+        return false;
+    }
+
+    jboolean return_value = env->CallBooleanMethod(object, method_id,
+            params[0].l);
+
+    return (bool) return_value;
+}
+
+bool AssimpImporter::callv(JNIEnv *env, jobject object, const char* type_name,
+        const char* method_name, const char* signature,/* const*/
+        jvalue* params) {
+    jclass java_class = env->FindClass(type_name);
```

Comment:
I think I personally know this is a wrapper method for CallVoidMethodA.  But just as we discussed before,  I think you'd better  not use the abbreviation name like this. Probably you can change it to something like callVoidMethod or callVoid.

Thanks in advance.